BackEnd/src/business/games.ts

```diff
@@ -201,4 +190,43 @@ export async function updatePlayerStatsForGame(matchId: string) {
       await updateStatsFor(playerGame, fullGame, playerGame.playerPuuid, number, playerGame.lobbyPosition as GameRoles, teamId);
     }
   }
+}
+
+async function saveMatchForGameAlreadyInDb(matchId: string) {
+  const existingObj = await GetDbGameByGameId(ToGameId(matchId));
+
+  if (!existingObj) {
+    return null;
+  }
+
+  // If its a game for a risen season then update the teams.
+  let recentlyBuiltRisenTeams = await hasRecentlyBuiltRisenTeams(existingObj.seasonId);
+  if (existingObj.seasonId && !recentlyBuiltRisenTeams) {
+    await buildRisenTeams(existingObj.seasonId);
+  }
+
+  const playerGames = await GetDbPlayerGamesByGameId(ToGameId(matchId));
+
+  if (playerGames.length !== 10) {
+    const foundPlayers = new Set<string>();
+    playerGames.map(pg => foundPlayers.add(pg.playerPuuid));
+    await UpdatePlayersInSingleMatchById(existingObj, await GetGameDataByMatchId(matchId), foundPlayers);
```

Comment:
Honestly I don't remember why this was added. Can we remove this and just say if there aren't 10 players we axe the game or something?

Reply:
So this case is valid for the scenario when a player hits the "update" button before we get a callback from riot. In this case the playergame would be generated for that one player instead of all 10. So i think we need to keep this. 

Looking at this, i think there is a bug here, i think we want the if statement to be < 10 so it will build the games if its not at 10 players